webservices/common/models/filings.py

```diff
@@ -12,7 +12,7 @@ class Filings(db.Model):
     candidate_id = db.Column(db.String, index=True, doc=docs.CANDIDATE_ID)
     candidate_name = db.Column(db.String, doc=docs.CANDIDATE_NAME)
     cycle = db.Column(db.Integer, doc=docs.RECORD_CYCLE)
-    sub_id = db.Column(db.BigInteger, primary_key=True)
+    sub_id = db.Column(db.BigInteger, index=True, primary_key=True)
```

Comment:
Is this the index you were referring to in the PR description, or do you mean in one of the `real_efile_*` tables?  Or both? :-)